README.md

```diff
@@ -41,9 +41,9 @@ M=MMMMMMMM++  FOR HUMANS   ++M8MMMMMM7M
 ## Build Requirements
 
 * OSX or Linux
-* GCC >= 5.0 or Clang >= 3.4 (with libc++)
-* CMake >= 3.0
-* Boost Libraries >= 1.57
+* GCC >= 4.8 or Clang >= 3.4 (with libc++)
+* CMake >= 3.2.2
+* Boost Libraries >= 1.54
```

Comment:
I don't think we do, but as far as I'm aware 1.54 is still sufficient. That's what the Facter docs currently say. We require 1.54 for Boost.Log.